templates/common/infra/bicep/core/networking/cdn.bicep

```diff
@@ -0,0 +1,38 @@
+// Module to create a CDN profile with a single endpoint
+param cdnProfileName string
+param cdnEndpointName string
+param location string = resourceGroup().location
+param tags object = {}
+
+@description('Origin URL for the CDN endpoint')
+param originUrl string
+
+@description('Delivery policy rules')
```

Comment:
Please sort alpha

Reply:
To clarify, should all params be alphabetized, or just ones with descriptions?
(I'm looking at other modules but I think they don't follow the style yet)

```
param cdnProfileName string
param cdnEndpointName string
param location string = resourceGroup().location
param tags object = {}

@description('Delivery policy rules')
param deliveryPolicyRules array = []

@description('Origin URL for the CDN endpoint')
param originUrl string
```

Reply:
Okay, I believe everything is alphabetized per preference. I've re-deployed a template with the files with no issue.